### PR TITLE
Types

### LINKED ISSUE #13: BE - Day Type
date
mood
name of habits completed

### PR DESCRIPTION
**Pull request by Lmcourt**
# Change Log

- Closes #13, 12, 9, 10, 11, 12

## What I did:

- Add types for habit, habit entry, journal, journal entry, and day
- update schema 
- update seed file
- Rebase

## Things to consider:

- MIGRATE + SEED
- No testing has been done

## Test coverage

- [] 100% Model Testing
- [] 100% Request Testing

If below 100%, what still needs testing:

- Everything

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,7 @@
...
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+FE do this for the DB:
+bundle install
+rails db:{drop,create,migrate,seed}
+rails s
+localhost:3001/graphiql
```
Comment (Lmcourt) [inline README.md:7]: Notes for FE to set up DB

**Review by WMudgeEllis — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (TannerDale) [inline README.md:7]: May need to add installing bundle with 'gem install bundler'

**Review by TannerDale — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (TannerDale) [reply README.md:7]: May need to add installing bundle with 'gem install bundler'

Comment (nzambonivergara) [reply README.md:7]: These are the commands I ran:

```
sudo gem install bundler
brew install rbenv
echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
source ~/.bash_profile
rbenv global 2.7.2
sudo bundle install
sudo gem install rails -v 5.2.6
sudo brew install postgresql
brew services start postgresql
rails db:{drop,create,migrate,seed}
rails s
```

**Review by woaishenzhen — COMMENTED** [review]
Great job!

**Pull request merged**